tests/test_urls.py

```diff
@@ -41,4 +42,5 @@ def test_url_include_without_namespace_fails(code):
     "re_path('/test/', includes('test2', namespace='test-2'))",
 ])
 def test_url_include_with_namespace_success(code):
-    assert len(run_check(code)) == 0
+    result = run_check(code)
+    assert not error_code_in_result('DJ05', result)
```

Comment:
could you add a new line at the end of the file? Thanks!

```suggestion
    assert not error_code_in_result('DJ05', result)

```

Reply:
done